ocaml/libs/tracing/tracing.ml

```diff
@@ -239,6 +241,36 @@ module Export = struct
   end
 
   module Destination = struct
+    module File = struct
+      let export ~trace_id ~span_json : (string, exn) result =
+        try
+          (* TODO: *)
```

Comment:
TODO... is there something missing here?

Reply:
@YanRachel

Reply:
@jameshensmancitrix

Reply:
[6be7576](https://github.com/xapi-project/xen-api/pull/4958/commits/6be7576d40f3f6984434efbbddb96fe6a95a9d8c)